spec/cassettes/Octokit_EnterpriseAdminClient_Users/_delete_user/deletes_a_user.json

```diff
@@ -0,0 +1 @@
+{"http_interactions":[{"request":{"method":"delete","uri":"<<ENTERPRISE_HOSTNAME>>admin/users/afakeperson","body":{"encoding":"UTF-8","base64_string":"e30=\n"},"headers":{"User-Agent":["Octokit Ruby Gem 4.1.1"],"Accept":["application/vnd.github.v3+json"],"Content-Type":["application/json"],"Authorization":["token <<ENTERPRISE_ACCESS_TOKEN>>"],"Accept-Encoding":["gzip;q=1.0,deflate;q=0.6,identity;q=0.3"]}},"response":{"status":{"code":204,"message":"No Content"},"headers":{"Server":["GitHub.com"],"Date":["Fri, 16 Oct 2015 16:25:44 GMT"],"Status":["204 No Content"],"X-Oauth-Scopes":["admin:org, admin:org_hook, admin:public_key, admin:repo_hook, delete_repo, gist, notifications, repo, user"],"X-Accepted-Oauth-Scopes":[""],"X-Github-Media-Type":["github.v3; format=json"],"X-Xss-Protection":["1; mode=block"],"X-Frame-Options":["deny"],"Content-Security-Policy":["default-src 'none'"],"Access-Control-Allow-Credentials":["true"],"Access-Control-Expose-Headers":["ETag, Link, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval"],"Access-Control-Allow-Origin":["*"],"X-Github-Request-Id":["d0cc7712-302e-4bb6-9204-b925d95e8931"],"Strict-Transport-Security":["max-age=31536000; includeSubdomains; preload"],"X-Content-Type-Options":["nosniff"]},"body":{"encoding":"UTF-8","base64_string":""},"http_version":null},"recorded_at":"Fri, 16 Oct 2015 16:25:44 GMT"}],"recorded_with":"VCR 2.9.3"}
```

Comment:
Since VCR records those, I think we've been [adding them as-is](https://github.com/octokit/octokit.rb/pull/618/files).